source/components/Select/styles.scss

```diff
@@ -0,0 +1,103 @@
+.select-cont {
```

Comment:
Instead of prepending every single class name with `select` take advantages of SASS's ability to handle nested CSS. Define the top level div to be select, and then on things such as the label, you can nest CSS like

```
.select {
  ...
  /* Styles for top level select component */

  label {
     ...
  }
}
```